README.md

```diff
@@ -16,6 +16,7 @@ The argument `'foo'` is short for `{foo: true}`. If the value of the key is fals
 
 ```js
 classNames('foo', 'bar'); // => 'foo bar'
+classNames('foo bar', 'bar'); // => 'foo bar'
```

Comment:
Whoops -- I was looking at two modules and I got them reversed. That was exactly what I was trying to document but I was wrong. Thanks!